modules/c/pages/gs-install.adoc

```diff
@@ -85,7 +85,7 @@ For Android and iOS the symbols are incorporated in the standard release package
 
 You can obtain the download for the Vector Search extension here:
 
-* _Vector Search Extension_ -- link:https://www.couchbase.com/downloads/?family=couchbase-lite#LiteExtensions[Couchbase Lite Extensions]
+* _Vector Search Extension_ -- xref:gs-downloads.adoc#vs-release-1-0-0-beta.3[Download Vector Search]
```

Comment:
hardcoded version?

Reply:
ah, and hardcoded links throughout this file?
I guess this is a Quick Fix to tidy later?

Reply:
Ah misunderstood. Quick fix to get it working correctly. Now correcting this and some other minor formatting issues in a separate PR